sql/mob_pools.sql

```diff
@@ -274,7 +274,7 @@ INSERT INTO `mob_pools` VALUES (218,'Archaic_Gear','Archaic_Gear',119,0x00001B07
 INSERT INTO `mob_pools` VALUES (219,'Archaic_Gears','Archaic_Gears',120,0x00001C0700000000000000000000000000000000,1,1,12,240,100,0,1,1,1,2,0,64,0,1153,4,0,0,0,0,120,120);
 INSERT INTO `mob_pools` VALUES (220,'Archaic_Mirror','Archaic_Mirror',238,0x0600150600000000000000000000000000000000,1,1,0,240,100,0,0,0,0,0,0,0,7,1155,4,0,0,0,0,0,238);
 INSERT INTO `mob_pools` VALUES (221,'Archaic_Rampart','Archaic_Rampart',209,0x00001A0700000000000000000000000000000000,1,1,7,320,100,0,1,0,1,2,0,0,0,1153,4,0,0,0,0,209,209);
-INSERT INTO `mob_pools` VALUES (222,'Archer_Pugil','Archer_Pugil',197,0x00005C0100000000000000000000000000000000,1,1,7,240,100,0,1,0,1,16,0,0,861,1155,0,0,0,0,0,197,197);
+INSERT INTO `mob_pools` VALUES (222,'Archer_Pugil','Archer_Pugil',197,0x00005C0100000000000000000000000000000000,5,4,7,450,100,0,1,0,1,16,0,0,861,0,0,0,0,0,0,412,197);
```

Comment:
From the videos you provided, both the sniper and archers attack every 4-5 seconds, based on mob movement.